app/src/main/java/com/abisoft/todocompose/ui/screens/AddTaskScreen.kt

```diff
@@ -34,16 +35,33 @@ import androidx.compose.ui.draw.alpha
 import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.res.colorResource
 import androidx.compose.ui.unit.dp
+import androidx.lifecycle.ViewModelProvider
+import androidx.lifecycle.viewModelScope
+import androidx.lifecycle.viewmodel.compose.viewModel
 import com.abisoft.todocompose.R
+import com.abisoft.todocompose.TodoItemsRepository
+import com.abisoft.todocompose.ui.viewModel.addTaskViewModel.AddTaskViewModel
+import com.abisoft.todocompose.ui.viewModel.addTaskViewModel.AddViewModelFactory
 import com.abisoft.todocompose.utils.DatePickerSwitch
 import com.abisoft.todocompose.utils.ImportanceSelection
+import com.example.myapplication.okhttp_client.ApiClient
+import com.example.myapplication.okhttp_client.TodoDto
+import kotlinx.coroutines.launch
+import okhttp3.Dispatcher
+import java.util.UUID
 
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
 fun AddTaskScreen(onTaskAdded: () -> Unit, onClose: () -> Unit) {
+    val apiService = ApiClient.create() // TodoApiService instansiyasini yaratish
```

Comment:
You should create 1 client per app lifecycle in most cases. Look explanation at https://github.com/sultan6233/todoapp/pull/2/files#diff-7a4f547403a7daeae93af7282c62fcb1edbdf17d2f963d4be4bf0833b5af9c58